packages/module/src/layouts/BaseLayout.ts

```diff
@@ -253,9 +252,9 @@ export class BaseLayout implements Layout {
       return undefined;
     }
 
-    let layoutNode = _.find(nodes, { id: node.getId() });
-    if (!layoutNode && _.size(node.getNodes())) {
-      layoutNode = _.find(nodes, { id: node.getChildren()[0].getId() });
+    let layoutNode = nodes.find(n =>  n.id === node.getId());
+    if (!layoutNode && node.getNodes().length) {
+      layoutNode = nodes.find(n => n.id === node.getChildren()[0].getId());
```

Comment:
IIRC for mobx performance, you'll want to cache the id instead of traversing the observables.
```suggestion
      const id = node.getChildren()[0].getId();
      layoutNode = nodes.find(n => n.id === id);
```